client/tomcatconf/commands.properties.in

```diff
@@ -485,6 +485,10 @@ updateVPCOffering=1
 deleteVPCOffering=1
 listVPCOfferings=15
 
+#### VPC OSPF config commands
+vpcOSPFConfig=15
+vpcOSPFConfigUpdate=1
```

Comment:
@abhinandanprateek the best practice is to add them to API annotation. There are few APIs using these annotations, but I agree that this is not yet popular among API authors.